docs/source/get_started/install.md

```diff
@@ -95,8 +97,46 @@ You should see an ASCII art graphic and the Kedro version number: for example,
 
 ![](../meta/images/kedro_graphic.png)
 
-If you do not see the graphic displayed, or have any issues with your installation, see the [frequently asked questions](../faq/faq.md), check out the [searchable archive from our retired Discord server](https://linen-discord.kedro.org), or post a new query on the [Slack organisation](https://slack.kedro.org).
+If you do not see the graphic displayed, or have any issues with your installation, see the [searchable archive of past community support discussions](https://linen-discord.kedro.org), or post a new query on [Kedro's Slack organisation](https://slack.kedro.org).
+
+### How do I upgrade Kedro?
+
+We use [Semantic Versioning](https://semver.org/). The best way to safely upgrade is to check our [release notes](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) for any notable breaking changes. Follow the steps in the migration guide included for that specific release.
```

Comment:
> We use Semantic Versioning

Do we? 😅 

> major version zero [means] Anything MAY change at any time

(from https://semver.org/#semantic-versioning-specification-semver)

I'd need someone from engineering to weigh in

Reply:
cc @merelcht on this. I will remove that first sentence anyway, since I don't think it adds anything to the docs at this point. Let's add it to the FAQ list though.